learning_assistant/api.py

```diff
@@ -16,6 +16,11 @@
     from common.djangoapps.course_modes.models import CourseMode
 except ImportError:
     CourseMode = None
+# todo: do we need try / catch for this?
+try:
+    from common.djangoapps.course_modes.models import CourseEnrollment
```

Comment:
I see that this wasn't done for the CourseMode import, but we have a file here https://github.com/edx/learning-assistant/blob/main/learning_assistant/platform_imports.py that can be used for platform imports. Using this pattern makes mocking these classes/functions a bit easier in unit testing.